platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -213,9 +215,11 @@ private void writeOutUsersDotPropertiesFile(File userPropertiesFile) {
         String oldHostValue = usersDotProperties.getProperty(oldHostName);
 
         if (oldHostValue != null) {
+          String newInternalHost = System.getProperty(SystemBaseUrl.INTERNAL_HOST);
+          String newDataManager = newInternalHost + "-" + DATA_MANAGER;
+          String newHostValue = oldHostValue.replaceAll(LOCALHOST_DATA_MANAGER, newDataManager);
```

Comment:
✏️I'd recommend getting rid of `newDataManager` and inlining it (and using String.format instead of concatenation):
`String newHostValue = oldHostValue.replaceAll(LOCALHOST_DATA_MANAGER, String.format("%s-%s", newInternalHost, DATA_MANAGER));`

Reply:
fixed